apps/researcher/src/lib/encode-uri-identifier.ts

```diff
@@ -0,0 +1,10 @@
+// Linked data uses URIs as identifiers.
```

Comment:
Consideration: remove this line? The frontend is/should be unaware of the use of Linked Data.